argo-cd-apps/overlays/base/smee/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
having smee in the base works for now since base is only imported in external overlays but this will not work when we start to deploy internal services or when when deploy internal clusters.

internal services is needed only on internal clusters and smee only on external clusters. 2 options either we put all applicationSet into  a folder and each overlay include explicitly which applicationset to include or we create a base-int and base-ext folder